src/nni_manager/yarn.lock

```diff
@@ -161,9 +161,10 @@
   version "10.5.2"
```

Comment:
@chicm-ms pls take a look. Seems that we should not update this file in this commit.

Reply:
I've undo the commit of this file and added it to .gitignore